src/cookbooks/crash_pings.md

```diff
@@ -0,0 +1,41 @@
+# Working with Crash Pings
+
+Here are some snippets to get you started querying [crash pings](../datasets/pings.md#crash-ping) with [STMO](../tools/stmo.md) and 
+[BigQuery](../cookbooks/bigquery.md). Using these tools, you can quickly get counts 
+and other information on the nature of crash pings submitted day-to-day.
+
+The simplest example is probably just to count all existing pings for a few days
+across some dimensions: 
+
+```sql
+SELECT date(submission_timestamp) AS crash_date,
+       count(*) AS crash_count
+FROM telemetry.crash
+WHERE date(submission_timestamp) >= DATE_SUB(CURRENT_DATE(), INTERVAL 7 DAY)
+GROUP BY crash_date
+```
+
+You can see an example of this query, along with a simple graph in [this STMO query](https://sql.telemetry.mozilla.org/queries/67925/).
```

Comment:
It looks like the graph is broken. X-axis doesn't plot the date.

Reply:
Good catch! I tweaked the query but forgot to update the visualization.